web/modules/custom/joinup_sparql/src/VocabularyFixturesReplace.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\joinup_sparql;
+
+use Drupal\Core\Database\Database;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use EasyRdf\Graph;
+use EasyRdf\GraphStore;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * JLA service modify vocabulary fixtures.
+ */
+class VocabularyFixturesReplace implements ContainerInjectionInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+  }
+
+  /**
+   * Replace current JLA list.
+   */
+  public function updatejla() {
```

Comment:
To be fair, it is a good idea to have a helper method like that. But we are duplicating work because we already have a command to import the fixtures, in `\Joinup\TaskRunner\Commands\VirtuosoCommands::importFixtures`.
IF we were to keep it, maybe we could make it configurable. Accept a parameter for the filepath and one parameter for the graph URI. Also, rename the method name to something more generic like `replaceFixturesInGraph`. Otherwise simply remove this and move the code to the update hook.